sdk/src/main/java/io/opentelemetry/sdk/common/Timestamp.java

```diff
@@ -35,7 +35,8 @@
   private static final long MAX_SECONDS = 315576000000L;
   private static final int MAX_NANOS = 999999999;
   private static final long MILLIS_PER_SECOND = 1000L;
-  private static final long NANOS_PER_MILLI = 1000 * 1000;
+  private static final long NANOS_PER_SECOND = 1000L * 1000L * 1000L;
```

Comment:
how about using `TimeUnit.SECONDS.toNanos(1)` and `TimeUnit.MILLISECONDS.toNanos(1)` for these?